cmd/jaeger/config-kafka-collector.yaml

```diff
@@ -35,5 +35,5 @@ exporters:
   kafka:
     brokers:
       - localhost:9092
-    topic: "jaeger-spans"
-    encoding: otlp_proto
+    topic: "env:KAFKA_TOPIC:-jaeger-spans"
```

Comment:
Does it not need ${} ?

Reply:
@yurishkuro It does. Thanks! I was wondering why the tests were failing.